libraries/SITL/SIM_Aircraft.h

```diff
@@ -114,7 +114,7 @@ class Aircraft {
         return mag_bf;
     }
 
-    float gross_mass() const { return mass + external_payload_mass; }
+    float gross_mass() const { return mass + external_payload_mass + sitl->payload; }
```

Comment:
why not add it to the existing functionality here? https://github.com/ArduPilot/ardupilot/blob/85d25b205f981bc6396df693899545cc666e0120/libraries/SITL/SIM_Aircraft.cpp#L961